lib/AST/ASTContext.cpp

```diff
@@ -1912,7 +1912,7 @@ bool ASTContext::canImportModule(ImportPath::Element ModuleName) {
 }
 
 ModuleDecl *
-ASTContext::getModule(ImportPath::Module ModulePath) {
+ASTContext::getModule(ImportPath::Module ModulePath, SourceLoc ImportFromLoc) {
```

Comment:
I think we don't need a separate `SourceLoc` here, because `ImportPath::Module` inherits a `getSourceRange()` method from its superclass `detail::ImportPathBase<Module>`.

Reply:
Good point! I updated the last commit.